### ISSUE TITLE
Sound Property on notification

### ISSUE THREAD

**Issue by solomon-okolie** [bytefish/FcmSharp#30]
How do i send sound field with notification. I see AndroidNotification has this field but fcmclient send does not accept AndroidNotification.  Adding the sound through ApnsConfig does not send the notification with sound

**Comment by bytefish** [comment]
The Library implements the FCM HTTP v1 API:

* https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#AndroidNotification

The ``AndroidNotification`` is a Property of the ``AndroidConfig``:

* https://github.com/bytefish/FcmSharp/blob/master/FcmSharp/FcmSharp/Requests/AndroidConfig.cs#L31

The ``AndroidConfig`` is a property of the ``Message``:

* https://github.com/bytefish/FcmSharp/blob/master/FcmSharp/FcmSharp/Requests/Message.cs#L18

Basically read through the FCM documentation linked above, which also is the data model of FcmSharp.
